src/OpenLoco/src/Jukebox.cpp

```diff
@@ -0,0 +1,148 @@
+#include "Jukebox.h"
+#include "Audio/Audio.h"
+#include "Config.h"
+#include "Date.h"
+#include "Environment.h"
+#include "Localisation/StringIds.h"
+#include <OpenLoco/Interop/Interop.hpp>
+#include <numeric>
+
+using namespace OpenLoco::Environment;
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Jukebox
+{
+    static loco_global<uint8_t, 0x0050D434> _currentSong;
+    static loco_global<uint8_t, 0x0050D435> _lastSong;
```

Comment:
We no longer use vanilla routines for any audio, so these variables can actually be integrated!
```suggestion
    static uint8_t _currentSong; // 0x0050D434
    static uint8_t _lastSong;    // 0x0050D435
```

Reply:
_lastSong is now used in both Audio.cpp and Jukebox.cpp, but I change that.
Should I apply this in Audio.cpp, or should that be done in a separate pull request?

Reply:
if its part of the refactor/move, then yeah you can apply it here as well